cve_bin_tool/checkers/gcc.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for gcc
+
+https://www.cvedetails.com/vulnerability-list/vendor_id-72/product_id-960/GNU-GCC.html
+
+"""
+from . import Checker
+
+
+class GccChecker(Checker):
+    CONTAINS_PATTERNS = [
+        r"gcc: fatal error: no input files",
+        r"compilation terminated.",
```

Comment:
This doesn't seem a good pattern since other compiler software may also contain this string.